server/middleware/authorize/authorizeRequest.js

```diff
@@ -18,6 +18,7 @@ const defaultConvertRequestToAction = (req) => {
 const authorizeActions = (subject, convertRequestToAction = defaultConvertRequestToAction) => (req, res, next) => {
   const action = convertRequestToAction(req)
   const authorized = req.ability.can(action, subject)
+  // console.log('authorizeActions', subject, action, req.ability, authorized)
```

Comment:
I'm not sure what your position on commented out code is but I'd usually recommend removing it

Reply:
Yes the rule is usually to remove it. However, this log line turns out to be often re-instated during debug of CASL as it shows the current sign in status and roles.